### PR TITLE
Azs s2s vpn doc improvement

### PR DESCRIPTION
**Pull request by dbrennand**
I have fixed a number of issues with this article.

The whole process has been tested from start to finish. Fixed broken images and placeholders.

Most likely there are still grammatical issues I have missed.

### CODE REVIEW

File: articles/azure/azs-how-create-S2S-VPN.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ author: Bailey Lawson
 
 toc_rootlink: Users
 toc_sub1: How To
-toc_sub2:
+toc_sub2: Create a VPN connection
 toc_sub3:
 toc_sub4:
-toc_title: Create a site-to-site connection
+toc_title: Create a site-to-site VPN connection - portal
 toc_fullpath: Users/How To/azs-how-create-S2S-VPN.md
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN.md]: Add the toc_sub2 to the toc_fullpath, so it becomes:
Users/How To/Create a VPN connection/azs-how-create-S2S-VPN.md

File: articles/azure/azs-how-create-S2S-VPN.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ author: Bailey Lawson
 
 toc_rootlink: Users
 toc_sub1: How To
-toc_sub2:
+toc_sub2: Create a VPN connection
 toc_sub3:
 toc_sub4:
-toc_title: Create a site-to-site connection
+toc_title: Create a site-to-site VPN connection - portal
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN.md]: For consistent, Portal should have an uppercase P

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 3]
```diff
@@ -0,0 +1,485 @@
+---
+title: How to create a site-to-site connection using the UKCloud Azure Stack portal | UKCloud Ltd
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: It's a bit long, but the title should probably be:
How to create a site-to-site connection between Azure Stack and Public Azure using the UKCloud Azure Stack portal

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+author: Shaion O'Niel
+reviewer: Daniel Brennand
+lastreviewed: 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a VPN connection
+toc_sub3:
+toc_sub4:
+toc_title: Create a site-to-site VPN connection between Azure Stack & Public Azure - portal
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Can we remove anything from this title to make it a bit shorter?
Create a VPN connection between Azure Stack and Public Azure - Portal
Create a site-to-site connection between Azure Stack and Public Azure - Portal
Create a S2S VPN connection between Azure Stack and Public Azure - Portal

Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: For consistency Portal should have an uppercase P

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 5]
```diff
@@ -0,0 +1,485 @@
...
+toc_fullpath: Users/How To/azs-how-create-S2S-VPN-azure-to-azurestack.md
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Add the toc_sub2 to the toc_fullpath, so it becomes:
Users/How To/Create a VPN connection/azs-how-create-S2S-VPN-azure-to-azurestack.md

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 6]
```diff
@@ -0,0 +1,485 @@
...
+toc_mdlink: azs-how-create-S2S-VPN-azure-to-azurestack.md
+---
+
+# How to create a site-to-site connection between Azure Stack and Public Azure using the UKCloud Azure Stack portal
+
+## Overview
+
+This article shows you how to use the Azure Stack and Public Azure portals to create a Site-to-Site VPN between your UKCloud for Microsoft Azure environment and Public Azure environment.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: I don't think site-to-site needs the uppercase S's

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 7]
```diff
@@ -0,0 +1,485 @@
...
+
+3. In the **Featured** section, select **Local network gateway**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to be updated once you have generated the public ip with the connection.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: public IP

Comment (suehighmoor) [reply articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: I'd rephrase this a little to:
This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address for now as you will generate the public IP later.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 8]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Create new local network gateway](images/azs-browser-create-Lnetwork-gateway.png)
+
+## Create the VPN connection in Azure Stack
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your public Azure VPN:
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: site-to-site

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 9]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - Select the location of the Azure resource.
+
+   - **Subnet Name** - The name of the first subnet within the virtual network.
+
+   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
+
+   - **DDoS protection** - Select the DDoS protection you would like, for more information please see the [Azure DDoS Protection Standard overview](https://docs.microsoft.com/en-gb/azure/virtual-network/ddos-protection-overview).
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Select the DDoS protection you would like. For more information, see the [Azure DDoS Protection Standard overview](https://docs.microsoft.com/en-gb/azure/virtual-network/ddos-protection-overview).

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 10]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Service endpoints** - This allows you to enable one or more service endpoints for more information please see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: This allows you to enable one or more service endpoints. For more information, see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 11]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Firewall** - This allows you to enable Azure Firewall, which is a managed cloud-based network security service that protects your Azure Virtual Network resources.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Select this option to enable Azure Firewall, which is a managed cloud-based network security service that protects your Azure Virtual Network resources.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 12]
```diff
@@ -0,0 +1,485 @@
...
+
+6. Click **Create**.
+
+7. After your virtual network has deployed, you can view it by clicking **All services** in the favourites panel, then selecting **Virtual networks** under the networking section.
+
+    ![Virtual networks button in all services](images/azs-public-browser-services-virtual-networks.png)
+
+8. Select your virtual network from the list.
+
+9. In the blade for your virtual network you can; view and monitor the virtual network, change its settings and perform diagnostics and troubleshooting.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: No semicolon needed here:
In the blade for your virtual network, you can view and monitor the virtual network, change its settings and perform diagnostics and troubleshooting.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 13]
```diff
@@ -0,0 +1,485 @@
...
+
+## Overview
+
+This article shows you how to use the Azure Stack and Public Azure portals to create a Site-to-Site VPN between your UKCloud for Microsoft Azure environment and Public Azure environment.
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in both the Azure Stack and Public Azure portals.
+
+## Create a virtual network in Azure Stack
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: I would create an extra level of headings here to organise the topic a bit more clearly.
 ## Set up VPN on the Azure Stack side

Then the following sections that provide steps for Azure Stack use the next level of headings (###)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 14]
```diff
@@ -0,0 +1,485 @@
...
+# How to create a site-to-site connection between Azure Stack and Public Azure using the UKCloud Azure Stack portal
+
+## Overview
+
+This article shows you how to use the Azure Stack and Public Azure portals to create a Site-to-Site VPN between your UKCloud for Microsoft Azure environment and Public Azure environment.
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in both the Azure Stack and Public Azure portals.
+
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: As this is a long article, it might be worth including an extra section here that provides an overview of the process:
1. Set up VPN on the Azure Stack side
2. Set up VPN on the Public Azure side
3. Update local gateway IPs
4. Verify the VPN connection

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 15]
```diff
@@ -0,0 +1,485 @@
...
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - The resource group cannot be changed if you are adding a connection for an existing peer.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Add new connection](images/azs-browser-add-connection.png)
+
+6. Click **OK**.
+
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Add the extra level of headings as before for Azure Stack

## Set up VPN on the Azure Stack side

Then the following sections that provide steps for Public Azure use the next level of headings (###)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 16]
```diff
@@ -0,0 +1,485 @@
...
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Name** - The name of the virtual network gateway.
+
+   - **Region** - The region to place the virtual network gateway in.
+
+   - **Gateway type** - The type of virtual network gateway to create.
+
+       - To connect with an existing ExpressRoute circuit, select "ExpressRoute".
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Use bold for **Express Route** not quotes

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 17]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Virtual Network** - This is the virtual network that you created earlier in Public Azure.
+
+   - **Public IP address** - The public IP address to assign to this virtual network gateway. Only dynamic public IP addresses are supported.
+
+       - Click **Create new**.
+
+       - In the **Public IP address name** field, enter a public IP address name.
+
+   - **Enable active-active mode** - The gateway SKU must be either **VpnGw1**, **VpnGw2**, **VpnGw3**, **VpnGw1AZ**, **VpnGw2AZ**, **VpnGw3Az** or HighPerformance (legacy SKU).
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: If you want to enable active-active mode, the gateway SKU must be one of: **VpnGw1**, **VpnGw2**, **VpnGw3**, **VpnGw1AZ**, **VpnGw2AZ**, **VpnGw3Az** or **HighPerformance** (legacy SKU)

(only use bold if these values show up somewhere in the UI - if it's just for information, no bold is needed)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 18]
```diff
@@ -0,0 +1,485 @@
...
+
+3. In the **Local network gateway** blade, click **Create**.
+
+    ![Create button in local network gateway blade](images/azs-public-browser-new-local-network-gateway-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to updated once you have generated the public IP with the connection.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: I'd rephrase this a little to:
This is the public IP address of the VPN device that you want Azure to connect to. Enter a dummy IP address for now as you will generate the public IP later.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 19]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Create new local network gateway](images/azs-public-browser-create-local-network-gateway-blade.png)
+
+5. Click **Create**.
+
+## Create the VPN connection in Public Azure
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your Azure Stack VPN:
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: site-to-site

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 20]
```diff
@@ -0,0 +1,485 @@
...
+
+1. Navigate to your virtual network gateway by clicking **All services**, then select **Virtual network gateways** under the networking section.
+
+    ![Navigate to the Virtual network gateway](images/azs-public-browser-services-virtual-network-gateway.png)
+
+2. Select your virtual network gateway from the list.
+
+3. In the *new* blade for your virtual network gateway, select **Connections** under the **Settings** section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Is it the *new* blade? I would have thought it is just the blade for the virtual network gateway (as you are selecting an existing one rather than creating a new one...

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 21]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Connection type** - Select **Site-to-site (IPSec)**.
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: In this instance (Public Azure), should this be:
This is your Microsoft Azure subscription.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 22]
```diff
@@ -0,0 +1,485 @@
...
+
+   - **Resource Group** - When adding a connection for an existing peer, the resource group can't be changed.
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Add new connection](images/azs-public-browser-add-connection-blade.png)
+
+6. Click **OK**.
+
+## Update the local Gateway IPs in Azure Stack and Public Azure
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: gateway should have a lowercase g

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 23]
```diff
@@ -0,0 +1,485 @@
...
+
+In order to establish the connection you will need to identify the public IPs that you created in Azure Stack and Public Azure, and then update both local gateways.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Just use To instead of In order to:
To establish the connection...

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 24]
```diff
@@ -0,0 +1,485 @@
...
+
+### Update the local gateway in Public Azure
+
+1. In the Azure Stack portal, navigate to the *connections* blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Connections* blade
*Networking* section

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 25]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![All services connections networking section](images/azs-browser-all-services-connections-networking.png)
+
+2. Select the VPN connection you created in the earlier stages.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Instead of "earlier stages" I think it would be clearer to specify the actual title of the section (and maybe even make it a link)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 26]
```diff
@@ -0,0 +1,485 @@
...
+
+9. Click **Save**.
+
+### Update the local Gateway IPs in Azure Stack
+
+1. In the Public Azure portal, navigate to the Connections blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![Connections button under all services in public Azure](images/azs-public-browser-services-connections-button.png)
+
+2. Select the VPN connection you created in the earlier stages.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Instead of "earlier stages" I think it would be clearer to specify the actual title of the section (and maybe even make it a link)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 27]
```diff
@@ -0,0 +1,485 @@
...
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Make a note of the public IP. You can find this in the *Virtual network gateway* section.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 28]
```diff
@@ -0,0 +1,485 @@
...
+
+1. In the Azure Stack portal, navigate to the *connections* blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![All services connections networking section](images/azs-browser-all-services-connections-networking.png)
+
+2. Select the VPN connection you created in the earlier stages.
+
+    ![VPN connections](images/azs-browser-vpn-connection-view.png)
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Make a note of the public IP. You can find this in the *Virtual network gateway* section.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 29]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![VPN connection public IP](images/azs-browser-site-to-site-vpn-connection-public-ip.png)
+
+4. Log in to the [Public Azure portal](https://portal.azure.com).
+
+5. In the Public Azure portal, navigate to the *Local Network Gateway* blade by clicking **All services** in the favourites panel, then selecting **Local network gateways** under the *networking* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: As the previous step instructs the user to log into the Public Azure portal, you don't need to specify Public Azure portal at the beginning of this step:
Navigate to the *Local Network Gateway* blade by clicking...

Comment (suehighmoor) [reply articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Networking* section

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 30]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![Azure Public virtual network button](images/azs-public-browser-services-virtual-network-gateway-button.png)
+
+6. Select the local network gateway that you created in the earlier steps.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Be specific (link to appropriate section)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 31]
```diff
@@ -0,0 +1,485 @@
...
+
+8. Change the **IP address** field to the IP taken from the Azure Stack connection.
+
+    ![Azure Public local gateway public IP address](images/azs-public-browser-local-gateway-ip.png)
+
+9. Click **Save**.
+
+### Update the local Gateway IPs in Azure Stack
+
+1. In the Public Azure portal, navigate to the Connections blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Networking* section

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 32]
```diff
@@ -0,0 +1,485 @@
...
+
+2. Select the VPN connection you created in the earlier stages.
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
+
+    ![Public IP of virtual network gateway in Public Azure](images/azs-public-browser-connection-ip.png)
+
+4. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
+
+5. In the Azure Stack portal, navigate to the *Local Network Gateway* blade by clicking **All services** in the favourites panel, then selecting **Local network gateways** under the *networking* section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Don't need to specify In the Azure Stack portal here:
Navigate to the *Local Network Gateway* blade by clicking...

Comment (suehighmoor) [reply articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Networking* section

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 33]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![Azure Stack portal local network gateway](images/azs-browser-local-network-gateway-all-services.png)
+
+6. Select the local network gateway that you created in the earlier steps.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Be specific (link to appropriate section)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 34]
```diff
@@ -0,0 +1,485 @@
...
+
+7. In the *local gateway* blade, under *Settings*, select **Configuration**.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Local network gateway* blade

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 35]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![Azure Stack local network gateway configuration](images/azs-browser-local-network-gateway-configuration.png)
+
+8. Change the *IP address* field to the IP taken from the Public Azure connection.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: **IP address** should be bold instead of italics

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 36]
```diff
@@ -0,0 +1,485 @@
...
+
+9. Click **Save**.
+
+## Verify the VPN connection
+
+After configuring the VPN device on your local network, you can verify the VPN connection with the following steps:
+
+1. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
+
+2. Navigate to your virtual network gateway by clicking **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Networking* section

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 37]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![Azure Stack virtual network gateways](images/azs-browser-virtual-network-gateway-all-services.png)
+
+3. Select your virtual network gateway from the list.
+
+4. In the blade for your virtual network gateway, select **Connections** under the **Settings** section.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: *Settings* section (italics, not bold)

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md [hunk 38]
```diff
@@ -0,0 +1,485 @@
...
+
+    ![Azure Stack virtual network gateway connections](images/azs-browser-virtual-network-gateway-connections.png)
+
+5. Select your connection from the list.
+
+6. In the *new* blade, you can view more information about your connection. The connection status will be **Connected** if there is a successful connection.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: In the blade for your connection, you can view more information. The connection status will be **Connected** if there is a successful connection.

File: articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md (hunk 21 above)
Comment (dbrennand) [reply articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md]: Changed to: This is your Public Azure subscription.

File: articles/azure/azs-how-create-S2S-VPN.md (hunk 1 above)
Comment (cblackuk) [reply articles/azure/azs-how-create-S2S-VPN.md]: Fixed
